opcua.cpp

```diff
@@ -5,14 +5,17 @@
  *
  * Released under the Apache 2.0 Licence
  *
- * Author: Mark Riddoch
+ * Author: Mark Riddoch, Massimiliano Pinto
  */
 #include <opcua.h>
 #include <reading.h>
 #include <logger.h>
+#include <map>
 
 using namespace std;
 
+// Hold subscription variables
+map<string, bool> subscriptionVariables;
```

Comment:
This need not be a map, just use a vector of subscribed variables. Less storage used as simpler container.